core/domain/user_query_services.py

```diff
@@ -105,21 +103,20 @@ def _save_user_query(user_query):
     """
     user_query.validate()
 
+    query_params = {
+        predicate['backend_attr']: getattr(
+            user_query.params, predicate['backend_attr'])
+        for predicate in constants.EMAIL_DASHBOARD_PREDICATE_DEFINITION
+    }
```

Comment:
Isn't there a more simple way to do this, something like `dict(user_query.params)`?

Reply:
Done.